fit.py

```diff
@@ -62,7 +62,8 @@ def give_reflect(x, y, f, para, para_names, mod, language):
              {"zh_CN": "{}项幂：", "en": " Power{}:"},
              {"zh_CN": "{}{}阶有理数：", "en": " Rat{}{}:"},
              {"zh_CN": "{}项正弦：", "en": " Sin{}:"},
-             {"zh_CN": "威布尔：", "en": "Weibull:"}]
+             {"zh_CN": "威布尔：", "en": "Weibull:"},
+             {"zh_CN": "威布尔：", "en": "Lorentzian:"}]

```

Comment:
Attempting to add a new fit function, I don't have access to the ui code though.